open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.POW

```diff
@@ -2416,46 +3072,44 @@ PROCEDURE ERRORHANDLER
 **  509 - Error getting new share rate
 **  510 - Could not calculate new maturity date
 **  511 - Error creating new share
-**  512 - Error creating new name record(s)
```

Comment:
Did you mean to remove Error Code 512 and 513? And if so does
open-subshare-json-contract.md need to be changed? Or will it be left alone in case you use 512 or 513 later?

were removed in new version:
512 - Error creating new name record(s)
513 - Transfer funding error

Reply:
While reviewing the code for these changes I discovered that error codes 512 & 513 were not being processed in the same manner as the other error codes. This is due to the fact that by the time a 512 or 513 occur, the share has already been created and thus, program halt cannot occur.  512 is being handled by passing the "created:[true/false]" being passed to the UX for each name creation attempt and code 513 is handled by passing the "successfulTransfer:[true/false]" value being passed to the UX.

Yes, the contract will need to be / should be updated.